NVDSuppressions.xml

```diff
@@ -183,33 +183,6 @@
     </notes>
```

Comment:
There are quite a few suppressions that can be removed now. See [here](https://dev.azure.com/ukhydro/Exchange%20Set%20Service/_build/results?buildId=376110&view=logs&j=cd3dc03b-3464-55ae-8318-cf74752c0973&t=6465cd95-d344-50dc-38f2-664baecded70&l=47).